src/coffee/directives/api/utils/fit-helper.coffee

```diff
@@ -9,5 +9,6 @@ angular.module('uiGmapgoogle-maps.directives.api.utils')
           everSet = true unless everSet
           point = if _.isFunction markerOrPoint.getPosition then markerOrPoint.getPosition() else markerOrPoint
         bounds.extend point
-      gMap.fitBounds(bounds) if everSet
+      if everSet
+        $timeout () -> gMap.fitBounds(bounds)
```

Comment:
OK, I will probably merge this. I just need to check some of my work I have in progress. I was working on a PR that would fix fit for most if not all singular directives as well. So I think i did make some changes to this function.